pkgs/games/katawa-shoujo/default.nix

```diff
@@ -0,0 +1,179 @@
+    # No autoPatchelfHook on Darwin
+    wrapProgram ${gameBin} \
+      --prefix DYLD_LIBRARY_PATH : ${lib.makeLibraryPath buildInputs}
+  '') + ''
+
+    # Delete binaries for wrong arch, autoPatchelfHook gets confused by them & less to keep in the store
+    find "$(dirname ${gameLibDir})" -mindepth 1 -maxdepth 1 \
+      -not -name 'python*' -not -name ${gameArch} \
+      -exec rm -r {} \;
+
+    # Replace some bundled libs so Nixpkgs' versions are used
+    rm ${gameLibDir}/libz*
+    rm ${gameLibDir}/libfreetype*
+    rm ${gameLibDir}/libSDL-1.2*
+  '' + lib.optionalString (!withSDL2) ''
+    # Only replace prebuild SDL_* libraries when using real SDL1
+    # Nixpkgs' SDL_compat + SDL_* libraries doesn't work
+    rm ${gameLibDir}/libSDL_image-1.2*
+    rm ${gameLibDir}/libSDL_ttf-2.0*
+  '' + lib.optionalString devendorImageLibs ''
+    rm ${gameLibDir}/libjpeg*
+    rm ${gameLibDir}/libpng12*
+  '' + ''
+
+    mkdir -p $out/share/{doc,licenses}/katawa-shoujo
+    mv ${gameDataDir}/'Game Manual'.pdf $out/share/doc/katawa-shoujo/
+    mv ${gameDataDir}/LICENSE.txt $out/share/licenses/katawa-shoujo/
+
+    mkdir -p $out/bin
+    ln -s ${gameBin} $out/bin/katawa-shoujo
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Bishoujo-style visual novel by Four Leaf Studios, built in Ren'Py";
+    longDescription = ''
+      Katawa Shoujo is a bishoujo-style visual novel set in the fictional Yamaku High School for disabled children,
+      located somewhere in modern Japan. Hisao Nakai, a normal boy living a normal life, has his life turned upside down
+      when a congenital heart defect forces him to move to a new school after a long hospitalization. Despite his difficulties,
+      Hisao is able to find friends—and perhaps love, if he plays his cards right. There are five main paths corresponding
+      to the 5 main female characters, each path following the storyline pertaining to that character.
+
+      The story is told through the perspective of the main character, using a first person narrative. The game uses a
+      traditional text and sprite-based visual novel model with an ADV text box.
+
+      Katawa Shoujo contains adult material, and was created using the Ren'Py scripting system. It is the product of an
+      international team of amateur developers, and is available free of charge under the Creative Commons BY-NC-ND License.
+    '';
+    homepage = "https://www.katawa-shoujo.com/";
+    # https://www.katawa-shoujo.com/about.php
+    # November 2022: Update, is it still ND?
+    # https://ks.renai.us/viewtopic.php?f=13&p=248149#p248149
+    license = with licenses; [ cc-by-nc-nd-30 ];
+    maintainers = with maintainers; [ OPNA2608 ];
+    # Building Ren'Py6 from source would allow more, but too much of a hassle
+    platforms = platforms.x86;
+    sourceProvenance = with sourceTypes; [ binaryNativeCode ];
+    # Needs different srcDetails & installPhase
+    broken = stdenv.hostPlatform.isWindows;
```

Comment:
```suggestion
```
Windows is such a low tier platform that it is kinda expected to be broken if not explicitly mentioned but not that important